mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -135,22 +135,21 @@ pub enum Error {
         name: String,
     },
     #[snafu(display(
-        "Failed to complete rebuild of child {} of nexus {}, reason: {}",
+        "Rebuild task not found for child {} of nexus {}",
         child,
         name,
-        reason,
     ))]
-    CompleteRebuild {
-        child: String,
-        name: String,
-        reason: String,
-    },
+    RebuildTaskNotFound { child: String, name: String },
     #[snafu(display(
-        "Rebuild task not found for child {} of nexus {}",
+        "Failed remove rebuild task {} of nexus {}",
```

Comment:
Comment should be "Failed **to** remove rebuild task"

Reply:
dohh, doone